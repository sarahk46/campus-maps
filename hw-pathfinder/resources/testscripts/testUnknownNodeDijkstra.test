# Find a path with Dijkstra's algorithm when
# one of the two nodes is not in the graph regardless of ordering.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0

FindPath simpleGraph A C
FindPath simpleGraph C A