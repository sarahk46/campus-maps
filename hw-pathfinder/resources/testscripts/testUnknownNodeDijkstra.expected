# Find a path with Dijkstra's algorithm when
# one of the two nodes is not in the graph regardless of ordering.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph

unknown node C
unknown node C